bash/runcbldemos

```diff
@@ -5,23 +5,27 @@ set -e              # exit on all errors
 cd $(dirname $0)    # set to directory of script
 cd ..
 
-bash/cblclg zcobol/demo/HELLO $1 $2 $3 $4 $5 $6 $7 $8 $9
+demopath=zcobol/demo
```

Comment:
No issue, just curious. Why did you add the demopath variable for the new version?